docs/build/reference/zc-conformance.md

```diff
@@ -22,36 +22,36 @@ Here are the **`/Zc`** compiler options:
 
 | Option | Behavior |
 |--|--|
-| [`/Zc:__cplusplus`](zc-cplusplus.md) | Enable the `__cplusplus` macro to report the supported standard (off by default). |
+| [`/Zc:__cplusplus[-]`](zc-cplusplus.md) | Enable the `__cplusplus` macro to report the supported standard (off by default). |
 | [`/Zc:__STDC__`](zc-stdc.md) | Enable the `__STDC__` macro to report the C standard is supported (off by default). |
-| [`/Zc:alignedNew`](zc-alignednew.md) | Enable C++17 over-aligned dynamic allocation (on by default in C++17). |
-| [`/Zc:auto`](zc-auto-deduce-variable-type.md) | Enforce the new Standard C++ meaning for **`auto`** (on by default). |
-| [`/Zc:char8_t`](zc-char8-t.md) | Enable or disable C++20 native `u8` literal support as `const char8_t` (off by default, except under **`/std:c++20`**). |
+| [`/Zc:alignedNew[-]`](zc-alignednew.md) | Enable C++17 over-aligned dynamic allocation (implied by **`/std:c++17`** or later). |
+| [`/Zc:auto[-]`](zc-auto-deduce-variable-type.md) | Enforce the new Standard C++ meaning for **`auto`** (on by default). |
+| [`/Zc:char8_t[-]`](zc-char8-t.md) | Enable or disable C++20 native `u8` literal support as `const char8_t` (implied by **`/std:c++20`** or later). |
```

Comment:
Hi, I sometimes like the 'implied by' approach, but it doesn't mean the same thing as the text it replaces. It indicates that the switch may be on when another switch is specified, but it loses the clarity that it is off in all other cases.  It replaces a statement that is true in all cases with a more ambiguous one that it is true in a specific case. I prefer the wording for /Zc:char8_t, for example. I'd rather not take the 'implied by' changes unless they add the extra clarity that they are off, otherwise.  Thank you for adding the optional [-] notes!

Reply:
My goal was to make these parentheticals consistent, which improves readability. I chose the "implied by" form because it was terser. But I don't mind changing it over to the form on this line.

Some cases are a bit different than others. If you still don't like the wording, let me know and I'll update it.

Reply:
I replaced "implied by" everywhere with the form on line 29.

I also made another tweak to the twoPhase row. That one is a trickier special case. I hope the new wording is clearer. I think the original wording was prone to misinterpretation.